lib/query.js

```diff
@@ -52,3 +56,11 @@ export const getValue = node => {
         return getValue(node.args[0])
     }
 }
+
+export const getNumerator = (node) => query.isNeg(node)
+    ? node.args[0].args[0]
+    : node.args[0]
+
+export const getDenominator = (node) => query.isNeg(node)
+    ? node.args[0].args[1]
+    : node.args[1]
```

Comment:
`getNumerator` and `getDenominator` should also be updated to handle the case where we're subtracting a negative fraction.